python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -62,16 +54,21 @@ def __ray_send__(
     tensors = gpu_object_store.get_object(obj_id)
 
     backend = collective.get_group_handle(communicator_meta.communicator_name).backend()
-    device = COLLECTIVE_BACKEND_TO_TORCH_DEVICE[backend]
 
     from ray.experimental.collective import get_tensor_transport_manager
+    from ray.experimental.collective.util import device_match_transport
```

Comment:
What's the reason to use local import in case we have already imported torch in this file?

Reply:
refined